### DISCUSSION TITLE
sasharesume.com - Portfolio Website (sasharesume)

### DISCUSSION THREAD

**Discussion by giscus** [TheBigSasha/sasharesume.com#27] [Giscus]
# sasharesume.com - Portfolio Website (sasharesume)

Kinda meta, isn't it? Learn all about the technology behind this website.

https://www.thebigsasha.com/works/sasharesume

**Comment by TheBigSasha** [comment]
Wow! Look at that comment support using discus! So easy to implement it’s crazy!
